src/webgpu/api/validation/encoding/beginComputePass.spec.ts

```diff
@@ -0,0 +1,133 @@
+export const description = `
```

Comment:
Is this part of this PR or something different?

Reply:
I think that is from merging in ToT. I will rebase and force update when I get back to my desk.